crds/AzureApplicationGatewayLoadDistributionPolicy.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azureapplicationgatewayloaddistributionpolicies.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
+  scope: Namespaced
+  names:
+    kind: AzureApplicationGatewayLoadDistributionPolicy
+    plural: azureapplicationgatewayloaddistributionpolicies
+    singular: azureapplicationgatewayloaddistributionpolicy
+    shortNames:
+      - agldp
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                targets:
+                  description: "An array of services that contains information for how the service should be used for load balancing in Application Gateway"
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      backend:
+                        type: object
+                        properties:
+                          service:
+                            description: "Service references a Service as a Backend."
+                            type: object
+                            properties:
+                              name:
+                                type: string
+                                description: "Name is the referenced service. The service must exist in the same namespace as the Ingress object."
+                              port:
+                                type: object
+                                description: "Port of the referenced service. A port name or port number"
+                                properties:
+                                  number:
+                                    type: integer
+                                  name:
+                                    type: string
+                      weight:
+                        description: "Weight specifies the proportion of HTTP requests forwarded to the backend."
+                        type: integer
+                        default: 1
```

Comment:
Add min and max:
```
minimum: 1
maximum: 1000
```

https://swagger.io/docs/specification/data-models/data-types/#numbers